torch/quantization/fx/quantization_patterns.py

```diff
@@ -830,7 +839,7 @@ def convert(self, quantizer: QuantizerCls, node: Node, load_arg: Callable,
             return quantizer.quantized_graph.node_copy(node, load_arg(quantized=False))
 
         # TODO: make helper functions for (torch.quint8, torch.qint8, None)
-        if dtypes in [(torch.quint8, torch.qint8, None)]:
+        if not is_reference and dtypes in [(torch.quint8, torch.qint8, None)]:
```

Comment:
optional readability nit: might be nice to restructure these if statements like this

```
if dtypes in [(torch.quint8, torch.qint8, None)]:
  if not is_reference:
    ...
  else:  # is_reference
    ...
else:
  assert dtypes in [(torch.float16, torch.float16, None)]
  if not is_reference:
    ...
  else:  # is_reference
    ...
```

this way the order is consistent

Reply:
I wasn't sure if it would be more readable to explicate the 4 modes or to split them with 2x2 branches. If that is considered more readable I can change it.